content/blog/2019-11-17-salut-bucharest.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Salut Bucharest
+author: sathya
+type: post
+date: 2019-11-17T04:51:13+00:00
+url: /2019/11/17/salut-bucharest/
+categories:
+  - Life
+tags:
+  - bucharest
+  - romania
+  - career
+  - immigration
+
+---
+
+
+It's been a while since I posted anything, so lets get this started again. As I had tweeted couple of weeks back, I have moved to Bucharest, Romania. 
+
+<!--more-->
+
+
+{{< tweet 1182251512334077952 >}}
+
+{{< tweet 1184008163546284032 >}}
+
+So some of the obvious questions would be:
+
+- Why Romainia?
```

Comment:
Is this a pun about Romania being the main thing in your life for the past few months? :P

Reply:
haha maybe I should keep it as here :P